microsoft-edge/toc.yml

```diff
@@ -1283,8 +1283,8 @@
           items:
             - name: Win32
               items:
-#               - name: WebView2 in Win32 apps
-#                 href: ./webview2/platforms/win32.md
+                - name: WebView2 in Win32 apps
+                  href: ./webview2/platforms/win32.md
 
                 - name: Win32 sample app
```

Comment:
Move GetStart tut article above the Sample article?
6x.
@captainbrosset 

`toc.yml`:
Internal preview: https://review.learn.microsoft.com/microsoft-edge/webview2/samples/webview2apissample?branch=pr-en-us-3340
External preview: https://github.com/MicrosoftDocs/edge-developer/blob/user/mikehoffms/overview-apis-jan-2025/microsoft-edge/toc.yml
Before/Live: https://learn.microsoft.com/microsoft-edge/webview2/samples/webview2apissample

Reply:
done